modules/proj4/docs/developer-guide/proj4-guide.md

```diff
@@ -0,0 +1,66 @@
+# Overview
+
+The `@math.gl/proj4` module provides support for conversion between major geospatial coordinate systems and projections used with computer maps, such as:
+
+- [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System) (World Geodetic System) coordinates.
+- [Web Mercator Projection](https://en.wikipedia.org/wiki/Web_Mercator_projection)
+- etc
+
+## Classes
+
+| Class                   | Description |
+| ---                     | --- |
+| `Proj4Projection`       | A projection class |
+
+## Usage
+
+Reproject WGS84 coordinates in another CRS
+```js
+import {Proj4Projection} from '@math.gl/proj4';
+
+const projection = new Proj4Projection({from: 'WGS84', to: '...'});
+
+const wgs84Position = [21, 78, 5000];
+const reprojectedPosition = projection.project(wgs84Position);
+```
+
+## Geospatial Coordinate Systems
+
+### Aliases
+
+Note that Proj4Projection allows aliases to be defined and comes with the following pre-installed aliases.
+
+| Alias | Resolves To |
+| ---                | --- |
+| 'WGS84'            | 'EPSG:4326' |
+| 'EPSG:3785'        | 'EPSG:3857' 'EPSG:4269' |
+| 'GOOGLE'           | 'EPSG:3857' 'EPSG:4269' |
+| 'EPSG:900913'      | 'EPSG:3857' 'EPSG:4269' |
+| 'EPSG:102113'      | 'EPSG:3857' 'EPSG:4269' |
```

Comment:
@ibgreen Also this table is wrong. I guess I can see from the bullet spacing how you'd read it this way, but `EPSG:3857` and `EPSG:4269` are different projections. The first is web mercator, the second is latitude longitude but on the NAD83 datum instead of the WGS84 datum.

![image](https://user-images.githubusercontent.com/15164633/90282590-da8f3600-de2b-11ea-99b9-7b60ae48d0f6.png)

So instead it should really be:

| Alias | Resolves To |
| ---                | --- |
| 'WGS84'            | 'EPSG:4326' |
| 'EPSG:4269'        | 'EPSG:4269' |
| 'EPSG:3785'        | 'EPSG:3857' |
| 'GOOGLE'           | 'EPSG:3857' |
| 'EPSG:900913'      | 'EPSG:3857' |
| 'EPSG:102113'      | 'EPSG:3857' |

Reply:
Ah yes, I did get confused. Do you think you can open a corrective PR? 

Also, we can probably remove the table from the typescript file now.